journal-of-the-american-adacemy-of-audiology.csl

```diff
@@ -0,0 +1,204 @@
+  <info>
+    <title>Journal of the American Academy of Audiology</title>
+    <title-short>JAAA</title-short>
+    <id>http://www.zotero.org/styles/journal-of-the-american-adacemy-of-audiology</id>
+    <link href="http://www.zotero.org/styles/journal-of-the-american-adacemy-of-audiology" rel="self"/>
+    <link href="http://www.zotero.org/styles/genome-biology-and-evolution" rel="template"/>
+    <link href="http://www.audiology.org/publications-resources/journal-american-academy-audiology/instructions-authors" rel="documentation"/>
+    <author>
+      <name>O'Brien Patrick, PhD</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="biology"/>
+    <category field="medicine"/>
+    <issn>1050-0545</issn>
+    <eissn>2157-3107</eissn>
+    <updated>2017-01-06T12:09:37+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="editor">
+    <names variable="editor">
+      <name delimiter=", " name-as-sort-order="all" sort-separator=", " initialize-with="">
+        <name-part name="family" text-case="capitalize-first"/>
+        <name-part name="given" text-case="capitalize-first"/>
+      </name>
+      <label form="short" prefix=", "/>
+    </names>
+  </macro>
+  <macro name="series-editor">
+    <names variable="original-author">
+      <label form="short" text-case="capitalize-first" suffix=" "/>
+      <name and="symbol" delimiter=", "/>
+    </names>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name name-as-sort-order="all" sort-separator=" " delimiter=", " delimiter-precedes-last="always" initialize-with=""/>
+      <label form="short" prefix=", "/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="text" delimiter-precedes-last="never"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if variable="DOI">
+        <text variable="DOI" prefix="doi:"/>
+      </if>
+      <else>
+        <choose>
+          <if variable="URL">
+            <text variable="URL"/>
+            <group prefix=" (" suffix=")">
+              <text term="accessed" text-case="capitalize-first" suffix=" "/>
+              <date variable="accessed">
+                <date-part name="month" suffix=" "/>
+                <date-part name="day" suffix=", "/>
+                <date-part name="year"/>
+              </date>
+            </group>
+          </if>
+        </choose>
+      </else>
+    </choose>
+  </macro>
+  <macro name="title">
```

Comment:
this entire macro doesn't do anything. Delete it and replace any `<text macro="title"/>` calls by `<text variable="title"/>`